### ISSUE TITLE
Position Paper: RChain should consider Google Ads for 'Promotion'. 

### ISSUE THREAD

**Issue by jasoncruzzy** [rchain/bounties#754]
Objectives of SEO(google ads) for RChain.

* Inform developers about RChain~SEO has a good range of publicity 
* With the RChain website on the 'google ads' it allows interested developers get vital information about RChain
* Stimulate brand preference for RChain amongst Blockchain developers 
* Awareness Campaign~ SEO will help constant top mind awareness  

Check here for more on 'google ads' to RChain 
_link to unsourced material removed --@dckc_

*Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely.*

_Estimated Budget of Task:_ **$[$40/1hr]**
_Estimated Timeline Required to Complete the Task:_ **[5hrs]**
_How will we measure completion?_ **[Read google docs)**

*See CONTRIBUTING.md for details on budget and reward process.*

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by pmoorman** [comment]
Hey @jasoncruzzy I think SEM marketing is something that's being taken care of by the marketing organisation primarily. Please check with @patrick727 or Ankur and see how you can help there.

The description texts you pulled from Wikipedia, and it's not entirely clear to me what exactly you want to do under this issue, and what you need the budget for.

I'm adding the needs-SMART-objective label, and removing branding (since it's marketing rather than branding).

**Comment by jasoncruzzy** [comment]
@pmoorman This is a research work analysing the possibilities of Search
Engine Marketing(SEM) for RChain......The objectives of this issue is to
give a discreet guide on SEM as a Proposal to RChain

**Comment by pmoorman** [comment]
oke, please sync with @patrick727 to make sure we're not doing double work here. I still find it hard to imagine exactly what a "discreet guid on SEM" would be worth to RChain.

I would recommend we work on clarifying a little bit better what exactly we want to produce here, and how it'll play into the wider strategy before we proceed.

**Comment by jasoncruzzy** [comment]
@Patrick727 kindly reply my personal message on discord, so i can some more
clarity on this

**Comment by jasoncruzzy** [comment]
@pmoorman collaborated with @patrick727 on the 'proposal' for SEO. He considers the #754 as a very good idea and open for consideration. @pmoorman knowing exactly what we want to produce, I will request you read the google docs(proposal), Thanks 👉 https://docs.google.com/document/d/1Hxpxduf6dEgESOGhdPtI5kD3y1xBJVhGSnFvfi_7nMw/edit?usp=drivesdk

**Comment by pmoorman** [comment]
awesome, great work @jasoncruzzy

**Comment by pmoorman** [comment]
Upon a deeper review, this entire "analysis" seems to be a bunch of plagiarism from Wikipedia, the Google AdWords sales page, and maybe a few other sources.

https://www.google.com/intl/ALL_in/adwords/benefits/ seems to be the primary sources, but really almost EVERYTHING seems to be copy-pasted shamelessly.

...

@jasoncruzzy I see you've voted a $470 budget for this month. Tell me.... how in the world is that justified?

**I'm putting in a slashing vote**, and I suggest we close this issue and forget about it as soon as possible.

_(P.S. @Cryptovideos has voted a 85% budget to himself on this issue. Is that for real, or a mix-up?)_

**Comment by jasoncruzzy** [comment]
@pmoorman as a guide you might need to always ask questions and make enquiries before taking 'personal' egoist decisions.

The document can not be said to be said to be 'Plagiarism' like you claim because. The document is more like a proposal to RChain Marketing team and I used their metrics to clearly define what the position of RChain will be.
I spoke with @patrick727 and he thought it was a good document but said it was not the right time to carry out SEO(google ads)  to avoid 'Spamming' and @patrick said he is working with Disney already for ads. 

On @crypto voting 85% for himself that was clearly a mistake and an error.

You are quite rude, egoistic and too temperamental to be a lead guide on a sensitive subject like MARKETING

**Comment by jasoncruzzy** [comment]
@ian-bloom  @jimscarver @dckc @kitblake It is highly worrisome and frustrating how @pmoorman addresses issues of Marketing with so much and ego mainly based on his opinion. If he is not satisfied with any work done he can actually say it politely or with less insults and address it properly without making the collaborator look dumb and silly for trying to make an input on RChain.
I thought RChain should be a decentralized system where people present their ideas as issues, it could be accepted or it might not be which is understandable. But as a lead guide he should be ready to address issues in an encouraging manner as some of this "SHAMELESS" issues still takes time and resources to put together. 
@pmoorman RChain should be seen as decentralized, Professional and not for expressing personal 'emotions'

**Comment by dckc** [comment]
@pmoorman has earned his authority. I see no insults.

**Comment by pmoorman** [comment]
@jasoncruzzy if you create a document that is almost entirely copy-pasted from other websites without mentioning this, and presenting it as if it was your own work, [than that's plagiarism](https://en.wikipedia.org/wiki/Plagiarism) to me.

I pasted the document text into [this plagiarism checker](https://smallseotools.com/plagiarism-checker/) just now, and found it to be 64% plagiarism, and 36% unique. Upon further inspection, the part that's 36% unique seems to actually ALSO have been copy-pasted, but the detector doesn't really detect it.

I guess you're free to raise issues if you think I do a bad job as a guide, but I'd focus on the fact that you're trying to pass off a shameless case of plagiarism as something worth $470.

**TL;DR:** I'm retaining my slashing votes.

**Comment by jasoncruzzy** [comment]
@pmoorman that is still unfair because I was clearly presenting the RChain on SEO(google ads) and I consciously used the 'google ads' parameters and  metrics, I clearly do not see this as plagiarism because I cannot add or subtract for 'google ads' on its own services. 'Google ads' is the SEO and their statistics and basics were necessary to put up the document, I don't think such document will be 'meaningful' without a direct reference.

2)  I said 'Analysing' 
> "A systematic examination and evaluation of data or
information, by breaking it into its component parts to uncover their interrelationships."
Also I stated it is a proposal put up for consideration, using the basics and  parameters of the subject cannot be said to be 'plagiarism' in a case as this. 

I appeal you kindly reconsider your 'Slashing' votes on this..... Thanks

**Comment by jasoncruzzy** [comment]
@AyAyRon-P  @pmoorman  I already changed the title of the issue. 
The document was referenced from google ads links and Wikipedia without the aim of 'plagiarism' but t  put the referenced links together to try and open considerations for SEM(google ads)  for RChain. 
 The idea of this issue was to put forth 'google ads' for consideration, It was never intended to be an enforcement document  but a 'bidding' document.
  Probably I was misunderstood and also I did not clarify myself properly. I have also reduced the budget(subject to further adjustment) of the issue,  which will only cover for the human resource cost,  as data charges, time, research, typing, energy consumption e. t. c
  Kindly reconsider this issue as it was not meant to be COPY-PASTE but to put up the  SEO(google ads)  for consideration,  I had to consciously use its parameters and basics.  I DO NOT TAKE THIE DOCUMENT AS A PERSONAL WRITE-UP

**Comment by allancto** [comment]
@jasoncruzzy @pmoorman this topic came up at the RAM meeting today. There are two separate discussions, plagiarism and slashing. 

@jasoncruzzy what is the purpose of this issue? when i read the document below my impression is that you've written a "position paper" suggesting that RChain consider using Google Adwords. In that case I would vote a small budget for it, but first the appearance of plagiarism needs to be addressed. 1. clearly identify what is YOUR work and what is quoted from google. One way to approach this is to simply include the link to the google adwords page rather than copy-pasting anything. Once you've included the link you can refer to sections of it which you find might be important. Make double sure NOT to create any impression that you are taking credit for any words written by others; you can use "" or indents or any other means to avoid even the hint of any confusion.

If it's a position paper suggesting that RChain consider SEM I'd consider that valid (although i suspect that the marketing people are already aware of SEM). You could change the title slightly to clarify the intent, something like 
> Position paper: RChain Should Consider Using Google Ads

@pmoorman @jasoncruzzy I'd also like to comment on the language used in these comments. The language seems to include personal characterization that is unnecessary. My guess is that if @pmoorman takes a bit more time to guess at the intent and suggests corrections, and if @jasoncruzzy takes a bit more time to guess at the intent and figure out why the notion of plagiarism came up and try to address that, that this conversation could have been much easier and more productive. I've seen other (totally unrelated) personal attacks in discord channels, and it's important- if we're serious about Cooperation at Scale- to keep our discussions civil so that people don't get discouraged from working together.

**Comment by dckc** [comment]
I just looked and the document still doesn't show which parts are quotes. **Always** cite your sources. Until the document is clear about which parts are quotes, we really can't have it as part of our work. In an earlier similar situation (#133) I replaced un-sourced material with TODO markers. I'm not available to take the time for that just now, so rather than slashing, I removed the link. I would vote $0 if that didn't get the work closer to 3 budget votes.

**Comment by jasoncruzzy** [comment]
@allanco @dckc I must commend your efforts and patience reviewing this issue.
@allanco the issue was meant to be a ''Position paper''(a report outlining google ads basics and parameters 'if' RChain considers its service). 
 My(@jasoncruzzy) initial idea was to act as a middleman(3rd party) suggesting the services of 'google ads' to RChain, doing that it was inevitable using the 'google docs' basics and parameters as seen in the document.
 My apologies for not including the references used in the document. The document was never intended to be a personal write-up but a proposal(google ads) put up for consideration to the Marketing team.

@dckc I have included the references to the document, check at the bottom of the document here👉 https://docs.google.com/document/d/1dYufWfP8oGUPzbIfsDfcUycR6Oi5_mkfBUXFNvksiRo/edit?usp=drivesdk

**Comment by kitblake** [comment]
@jasoncruzzy This is still the case (here I'm going to quote what @dckc wrote above):

"I just looked and the document still doesn't show which parts are quotes. Always cite your sources."
– Source: @dckc in https://github.com/rchain/bounties/issues/754#issuecomment-402558960

A reader cannot see what parts are quotes, and the references don't indicate that either. We must follow academic standards by putting quotes in quotes, noting sources, and/or using footnotes.

**Comment by jasoncruzzy** [comment]
@kitblake I will make some readjustments I think I got a clearer view from
you👍👍

**Comment by jasoncruzzy** [comment]
@kitblake @AyAyRon-P  @allancto  @dckc I included  the "quotes and sources"  in the document  as requested  check it out  https://docs.google.com/document/d/1dYufWfP8oGUPzbIfsDfcUycR6Oi5_mkfBUXFNvksiRo/edit?usp=drivesdk

**Comment by pmoorman** [comment]
@jasoncruzzy I have re-reviewed this issue today, and decided to retain my slashing vote.

The reason is that — even after lengthy discussion about plagiarism with all the people above — you still fail to disclose what's actually going on. The very first paragraph of your document almost entirely matches the first paragraph of the [Wikipedia article](https://en.wikipedia.org/wiki/Search_engine_marketing) on the matter!!!

---

_I think that the document doesn't constitute any value to the RChain Coop, and that the plagiarism actively damages us. Anyone who wants to see the Bounty Program (of which I'm very proud) disappear needs only to collect 2-3 cases like this that went unpunished and conclude that the entire thing is rotten to the bone._

**Comment by jasoncruzzy** [comment]
I had read and learnt read about Search Engine Marketing via Wikipedia and 'Google ads' and I had defined the first "paragraph" with the basics of information I got from Wikipedia but certainly not(word for word). I am no longer interested in the budgeting of this issue as it seems not 'feasible'. But @pmoorman I'm yet to understand why the SLASHING VOTE on an issue that is yet to get a single budget or reward votes, clearly emotional @pmoorman.
   I'm very appreciative of the  efforts of @dckc  @allancto  @kitblake @AyAyRon-P  to guide me on this, I have a better knowledge of writing articles for next time. 
 @pmoorman  Please upon a deeper review I think you should study the role of a lead guide.

**Comment by pmoorman** [comment]
Final update (from my side, at least):

- The reason for the slash vote I've outlined above. Because I belief this issue hurts the RChain Coop, and it's bounty system.

- After discussion with other guides (specifically @allancto ), I belief there's no bad intent at play on this issue, mere misunderstanding. We came to the heuristic that slashing is for (suspected) bad intentions, and 0-voting for issues that don't constitute positive value but don't have bad intent.

- **I've now updated my slash vote to a 0-vote,** which means you'll get rewards for the other work you've done this month.

- I'll close this issue.
